nannou_egui_demo_app/src/main.rs

```diff
@@ -42,12 +45,12 @@ fn update(app: &App, model: &mut Model, update: Update) {
         ..
     } = *model;
     egui.set_elapsed_time(update.since_start);
-    let proxy = app.create_proxy();
-    egui.do_frame_with_epi_frame(proxy, |ctx, frame| {
-        egui_demo_app.update(&ctx, frame);
-    });
+    let ctx = egui.begin_frame();
```

Comment:
These should probably just be moved into a `do_with_frame` method on `egui` to keep it simple.